.pre-commit-config.yaml

```
@@ -0,0 +1,20 @@
+repos:
+  - repo: https://github.com/psf/black
+    rev: 24.4.2
+    hooks:
+      - id: black
+
+  - repo: local
+    hooks:
+      - id: pylint
+        name: pylint
+        entry: pylint
+        language: system
+        types: [python]
+        require_serial: true
+        args:
+          [
+            "-rn",
+            "-sn",
+            "--rcfile=.pylintrc"
```

For tooling, I always think it's preferable to use the long form of options, e.g., something like:

```suggestion
            "--reports=n",
            "--score=n",
            "--rcfile=.pylintrc"
```